### ISSUE TITLE
Should sound be turned on?

### ISSUE THREAD

**Issue by chrisklus** [phetsims/number-play#115]
@Luisav1 and I turned on sound for the game (and consequently, the rest of the sim) over in https://github.com/phetsims/number-play/issues/109. The sounds for the games all match what Fourier's game does, except the correct and incorrect buttons in this game are just standard button sounds. We should be able to easily update these to match the sounds that Fourier's game makes when an answer is submitted.

If we like having the sound on, this also may elevate the priority for the sound design for the rest of the sim.

**Comment by amanda-phet** [comment]
We decided not to turn on UI sounds for this sim (but game sounds seem like a good idea to me- I can review those separately). If/when the grant gets funded to work on this sim more, we will do sound design for the rest of the sim.

**Comment by chrisklus** [comment]
I would like to revisit this issue RE the title I just changed it to. Sound has been on for a while and no one has mentioned it - should we turn it of and only allow voicing? There will be no sound control to disable voicing if so.

**Comment by chrisklus** [comment]
In design meeting, we decided to turn off all sounds except the right and wrong answer feedback in the game, as well as entering and exiting the game level. So we will not turn off sound globally so there's still a mute button in the sim that will apply to voicing as well.

**Comment by chrisklus** [comment]
Today I got assistance from @pixelzoom and discussed with @amanda-phet - @amanda-phet and I decided to take the equality-explorer approach of turning `user-interface` sound volume to 0. This can be done in one place, and then the correct/incorrect answer sounds in the game still work.

To do what was requested in https://github.com/phetsims/number-play/issues/115#issuecomment-1029411686 would've required turning off sounds for likely dozens of individual controls, which doesn't seem worth it to gain two button sounds that were somewhat arbitrarily chosen as more important as something like a ResetAllButton sound, for example. We decided to wait til full sound design before enabling any user interface controls. Closing.
